ichnaea/data/tests/test_export.py

```diff
@@ -265,12 +265,32 @@ def test_upload(self, celery, session, stats):
         gotten = [report["position"]["accuracy"] for report in send_reports]
         assert set(expect) == set(gotten)
 
-        stats.check(
-            counter=[
-                ("data.export.batch", 4, 1, ["key:backup"]),
```

Comment:
The 4 here means that it's checking for 4 separate records like this. I converted this to a `len` of a `filter_records()` call.